superduperdb/misc/superduper.py

```diff
@@ -19,23 +28,33 @@ def run(cls, item: t.Any, **kwargs) -> t.Any:
 
     @classmethod
     def accept(cls, item: t.Any) -> bool:
-        count = cls.count or len(cls.attrs)
-        return sum(hasattr(item, a) for a in cls.attrs) == count
+        """Does this item match the DuckType?
+
+        The default implementation returns True if the number of attrs that
+        the item has is exactly equal to self.count.
+
+        """
+        return sum(hasattr(item, a) for a in cls.attrs) == cls.count
 
     @classmethod
     def create(cls, item: t.Any, **kwargs) -> t.Any:
+        """Create a superduperdb container for an item that has already been accepted"""
         raise NotImplementedError
 
+    _DUCK_TYPES: t.List[t.Type] = []
+
+    def __init_subclass__(cls, **ka):
```

Comment:
Nitpick: Is `**ka` a thing?

Reply:
Yes, you see it in new code fairly often in this idiom:

```
@functools.wraps(fn)
def wrapped(*a, **ka):
    ....
    return fn(*a, **ka)
```

where the arguments are covered up by the `wraps`.

I don't think this case is quite the same, and more, it breaks [the principle of least astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment), so I switched it to `kwargs`.